frontend/src/scenes/events/Events.tsx

```diff
@@ -34,10 +35,29 @@ const eventsLogic = kea<eventsLogicType>({
     }),
 })
 
+function UsageDisabledWarning(): JSX.Element {
+    return (
+        <Alert
+            type="warning"
+            message={
+                <>
+                    Event usage is not enabled on your instance. If you want to enable event usage please set the follow
```

Comment:
Should we say Event _Property_ usage is not enabled? Or does this setting disable events as well

Reply:
Yeah it should change depending on which tab you're on